src/error.rs

```diff
@@ -3,14 +3,16 @@
 use std::{array::TryFromSliceError, fmt::Debug, num::TryFromIntError};
 
 use cosmian_crypto_core::CryptoCoreError;
+#[cfg(feature = "hybrid")]
+use pqc_kyber::KyberError;
 use thiserror::Error;
 
 #[derive(Error, Debug)]
 pub enum Error {
     #[error("Unknown partition {0}")]
     UnknownPartition(String),
     #[error("{0}")]
-    CryptoError(CryptoCoreError),
+    CryptoError(String),
```

Comment:
Could we keep the typed `CryptoCoreError` error here?

Reply:
ok